blocking-elements.ts

```diff
@@ -0,0 +1,407 @@
+/**
+ * @license
```

Comment:
license currently not included in the dist/* files

Reply:
Fixed. For some reason the rollup license-preserving text wasn't working. I upgraded to the latest rollup and babel, removed rollem (not needed anymore), and also tweaked the script to only emit each unique license once (since there are now two input files).